OnLand/ControlLaptop/static/assets/js/dynamic.js

```diff
@@ -1,12 +1,41 @@
-function getStr(_class,_id) {
-    return ('.'+_class+' .'+_id+'').toString();}
-
-
-function sensorRefresh() {
-    // TBD: Wire up SocketIO and unload sensor values to table
+function sendCommand(command) {
+  $.ajax({
+    type: "POST",
+    url: '/send_command',
+    dataType: 'json',
+    contentType: 'application/json',
+    data: JSON.stringify({
+      command: command
+    }),
+    success: function HANDLE_COMMAND_SEND_SUCCESS (data) {
+    },
+  })
 }
 
+function registerDisconnectFromPodClick() {
+  $('#disconnect').on('click', function sendESTOPCOMMAND() {
+    $.ajax({
+      type: "POST",
+      url: '/disconnect_from_pod',
+      dataType: 'json',
+      contentType: 'application/json',
+      data: JSON.stringify({
+        command: 'DISCONNECT'
+      }),
+      success: function HANDLE_DISCONNECT_SUCCESS (data) {
```

Comment:
Same as above, mia error handling? (again I could be wrong I haven't done much ajax)